### ISSUE TITLE
Breaking change in version 1.2.1

### ISSUE THREAD

**Issue by astraldawn** [css-modules/postcss-icss-values#62]
```
ERROR in ./~/css-loader?modules&importLoaders=1&localIdentName=[path]___[name]__[local]___[hash:base64:5]!./src/components/Uploader/Uploader.css
Module build failed: Error: [object Object] is not a PostCSS plugin
```

Getting this error in 1.2.1. Resolved by downgrading to 1.1.3. Used together with css-loader.

**Comment by sorrycc** [comment]
Same problem with me, there is no `module.exports = exports['default'];` in `lib/index.js`.

**Comment by geelen** [comment]
Pushed the `latest` tag back to `v1.1.3` until this is fixed.

**Comment by guox191** [comment]
＋1

**Comment by baoti** [comment]
Fixed by 1.2.2.
